pkg/apis/build/v1alpha1/build_types.go

```diff
@@ -118,6 +118,8 @@ type BuildSpec struct {
 	// (`.spec.source`) data.
 	//
 	// +optional
+	//
+	// NOTICE: Multiple sources in a build are deprecated. This feature will be removed in a future release.
```

Comment:
It seems to work based on what I see being generated. For consistency, I would always have the "annotations" at the end.

```suggestion
	// NOTICE: Multiple sources in a build are deprecated. This feature will be removed in a future release.
	//
	// +optional
```